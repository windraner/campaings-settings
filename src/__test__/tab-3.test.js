import React from 'react';
import Tab3 from '../components/main-content-section/tab-block/tab-3/tab-3';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';

configure({ adapter: new Adapter() });

const mock = {
  userBlackList: ['asd', 'fgh'],
  hashtagBlackList: ['zxc', 'vbn'],
  stateHandler() {},
};

describe('<Tab3 />', () => {
  it('matches the snapshot', () => {
    const tree = shallow(<Tab3 {...mock} />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
