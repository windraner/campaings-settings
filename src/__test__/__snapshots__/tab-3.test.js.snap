// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Tab3 /> matches the snapshot 1`] = `
<Fragment>
  <TabTitle
    text="Blacklisting"
  />
  <TabContainer>
    <CreatableSelectInput
      field="userBlackList"
      itemsList={
        Array [
          "asd",
          "fgh",
        ]
      }
      stateHandler={[Function]}
      text="Do not collect media of these users (max. 3, comma-separated)"
    />
    <div
      className="campaigns-settings__divider-mt-32"
    />
    <CreatableSelectInput
      field="hashtagBlackList"
      itemsList={
        Array [
          "zxc",
          "vbn",
        ]
      }
      stateHandler={[Function]}
      text="Do not include media with hashtag (max. 3, comma-separated)"
    />
  </TabContainer>
</Fragment>
`;
